lldb/tools/lldb-dap/Protocol/ProtocolRequests.cpp

```diff
@@ -262,6 +262,37 @@ json::Value toJSON(const BreakpointLocationsResponseBody &BLRB) {
   return json::Object{{"breakpoints", BLRB.breakpoints}};
 }
 
+bool fromJSON(const json::Value &Params, Console &C, json::Path P) {
+  auto oldFormatConsole = Params.getAsBoolean();
+  if (oldFormatConsole) {
+    if (*oldFormatConsole)
+      C = eConsoleIntegratedTerminal;
+    else
+      C = eConsoleInternal;
```

Comment:
Nit: you could use ternary operator here.
```suggestion
    C = *oldFormatConsole ? eConsoleIntegratedTerminal : eConsoleInternal;
```